src/analyzer/protocol/dce-rpc/dce_rpc-auth.pac

```diff
@@ -24,24 +26,43 @@ refine connection DCE_RPC_Conn += {
 			ntlm->Done();
 			delete ntlm;
 			}
+		if ( krb )
+			{
+			krb->Done();
+			delete krb;
+			}
+
 	%}
 
 	function forward_auth(auth: DCE_RPC_Auth, is_orig: bool): bool
 		%{
-		switch ( ${auth.type} )
+		switch ( ${auth.type} )  // https://social.msdn.microsoft.com/Forums/en-US/44212c32-a4f6-4960-8799-0e00821650f4/msrpc-and-dcerpc-security?forum=os_windowsprotocols
 			{
 			case 0x09:
 				if ( ! gssapi )
-					gssapi = zeek::analyzer_mgr->InstantiateAnalyzer("KRB", zeek_analyzer()->Conn());
+					gssapi = zeek::analyzer_mgr->InstantiateAnalyzer("GSSAPI", zeek_analyzer()->Conn());
 				if ( gssapi )
 					gssapi->DeliverStream(${auth.blob}.length(), ${auth.blob}.begin(), is_orig);
 				break;
+			case 0x10:
+				if ( ! krb )
+					krb = zeek::analyzer_mgr->InstantiateAnalyzer("KRB", zeek_analyzer()->Conn());
+				if ( krb )
+					krb->DeliverStream(${auth.blob}.length(), ${auth.blob}.begin(), is_orig);
+				break;
+
 			case 0x0a:
 				if ( ! ntlm )
 					ntlm = zeek::analyzer_mgr->InstantiateAnalyzer("NTLM", zeek_analyzer()->Conn());
 				if ( ntlm )
 					ntlm->DeliverStream(${auth.blob}.length(), ${auth.blob}.begin(), is_orig);
 				break;
+                        case 0x44:
+				zeek_analyzer()->Weird("netlogon_dce_rpc_auth_type", zeek::util::fmt("%d", ${auth.type}));
+				break;
+                        case 0x0E:
+				zeek_analyzer()->Weird("tls_dce_rpc_auth_type", zeek::util::fmt("%d", ${auth.type}));
+				break;
```

Comment:
Are you seeing these in practice?

Reply:
Added a netlogon pcap and test. I have no pcap from the tls auth.

Reply:
Thanks for the tests! My question was also aiming at log volume: do these new weirds tend to show up frequently?

Reply:
The volume of the weird.log doesn't change. Currently the 0x44 and 0x0E and unmatched are sent to the logs with:

```
	default:
		zeek_analyzer()->Weird("unknown_dce_rpc_auth_type", zeek::util::fmt("%d", ${auth.type}));
		break;
```
The cases in this PR just make the names more descriptive, since it's not really unkown. Based on experience I would say that the Netlogon is quite common, perhaps not even worth showing up in the weird.log.

Reply:
Ok, thanks.